sarxarray/_io.py

```diff
@@ -13,6 +14,60 @@
 # Example: https://docs.dask.org/en/stable/array-creation.html#memory-mapping
 
 
+def from_zarr(files_zarr: str | Path) -> xr.Dataset:
```

Comment:
The `from_zarr` function currently wraps the [`xarray.open_zarr`](https://docs.xarray.dev/en/latest/generated/xarray.open_zarr.html) function but does not expose its additional arguments, which restricts users' flexibility when using `from_zarr`. I suggest adding `**kwargs` to allow users to specify other arguments for `open_zarr`, or at least, expose some commonly used parameters e.g. `chunks`, `drop_variables`, and `zarr_version`, ....